apps/api/src/app/controllers/__tests__/__snapshots__/userManagementQueries.e2e.spec.ts.snap

```diff
@@ -274,36 +274,36 @@ Object {
 `;
 
 exports[`User Management Queries when the system user issuing the command is authorized when the user is a coscrad admin when querying for user groups when fetching all groups should return the expected result 1`] = `
-Object {
-  "entities": Array [
-    Object {
-      "actions": Array [
-        Object {
+{
```

Comment:
I'm guessing not having the `Object` and `Array` notations is the desired result?